mcp_server/quantum_tools.py

```
@@ -21,437 +21,476 @@
         """Convert LLM training problem to QUBO optimization"""
         # This is a simplified conversion - real implementation would be more sophisticated
-        
+
         # Extract training parameters
-        learning_rate = model_config.get('learning_rate', 0.001)
-        batch_size = model_config.get('batch_size', 32)
-        epochs = model_config.get('epochs', 10)
-        
+        learning_rate = model_config.get("learning_rate", 0.001)
+        batch_size = model_config.get("batch_size", 32)
+        epochs = model_config.get("epochs", 10)
```

Ruff (0.11.9)

Local variable `epochs` is assigned to but never used

Remove assignment to unused variable `epochs`

(F841)